backend/tilavarauspalvelu/integrations/email/typing.py

```diff
@@ -44,3 +74,413 @@ def build(
     def __json__(self) -> dict[str, Any]:  # noqa: PLW3201
         """Make the object JSON serializable to be used in Celery tasks."""
         return asdict(self)
+
+
+# ruff: noqa: N801
+class EmailType:
```

Comment:
Just a note here about our discussion about this class. Mark this as done when done or ask questions if need be.